cli/api_client.go

```diff
@@ -0,0 +1,221 @@
+package cli
```

Comment:
I would not create separate package, but add next to `cmd/mysterium_client/command_interactive`